src/launcher/features/apps/App/app.module.scss

```diff
@@ -12,7 +12,7 @@
 
 @keyframes appWasInstalled {
     0% {
-        background-color: $primary;
+        background-color: #F0FBFC;
```

Comment:
will move color to the shared after approval

Reply:
maybe add a note that this color is only used on exceptions?

Reply:
Added in a single place https://github.com/NordicSemiconductor/pc-nrfconnect-shared/pull/968